src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -569,15 +578,14 @@ static void WindowLoadsaveTextinput(rct_window* w, rct_widgetindex widgetIndex,
     switch (widgetIndex)
     {
         case WIDX_NEW_FOLDER:
+        {
             if (!filename_valid_characters(text))
             {
                 context_show_error(STR_ERROR_INVALID_CHARACTERS, STR_NONE, {});
                 return;
             }
 
-            safe_strcpy(path, _directory, sizeof(path));
-            safe_strcat_path(path, text, sizeof(path));
-
+            const u8string path{ Path::Combine(_directory, text) };
```

Comment:
Any reason to use this way of initialization instead of simple assignment? It's just a single value, and with assignment can't be confused with list initialization.